dist/templates/ovnkube-node.yaml.j2

```diff
@@ -259,6 +259,13 @@ spec:
 
       nodeSelector:
         kubernetes.io/os: "linux"
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: nvidia.com/smart-nic
```

Comment:
Do we want to define a general label using ovn-k8s prefix?

Reply:
yes we do. this is actually in my TODO list.

i will update the prefix to `k8s.ovn.org`